internal/jwks/jwks_test.go

```diff
@@ -256,3 +257,64 @@ func TestStrip(t *testing.T) {
 		}
 	}
 }
+
+func TestUpdateAll(t *testing.T) {
+	t.Parallel()
+
+	docContents := encodeKeys(key1, key2)
+
+	cases := []struct {
+		name  string
+		delay time.Duration
+		err   string
+	}{
+		{
+			name:  "successful",
+			delay: time.Duration(0),
+			err:   "",
+		},
+		{
+			name:  "failure",
+			delay: 2 * time.Second,
+			err:   "failed to processes ha.1: reading connection",
+		},
+	}
+
+	for _, tc := range cases {
+		//scopelint:ignore
```

Comment:
Don't ignore that - that linter is actually telling you that something is wrong 😄 

You need `tc := tc` here. Otherwise you might run the same test N times where N is the last value in the range.